src/build-actions.js

```diff
@@ -202,28 +206,37 @@ const prepareToBuildAction = async (zipFileName, action, root, dist) => {
  *  By default only actions which were not built before will be zipped.
  *  Last built actions data will be used to validate which action needs zipping.
  *
- * @param {Array<ActionBuild>} buildsList Array with data about actions available to be zipped.
+ * @param {Array<ActionBuild>} buildsList Array of data about actions available to be zipped.
  * @param {string} lastBuildsPath Path to the last built actions data.
- * @param {boolean} skipCheck when true will zip all the actions from the buildsList
+ * @param {string} distFolder Path to the output root.
+ * @param {boolean} skipCheck If true, zip all the actions from the buildsList
  * @returns {string[]} Array of zipped actions.
  */
-const zipActions = async (buildsList, lastBuildsPath, skipCheck) => {
+const zipActions = async (buildsList, lastBuildsPath, distFolder, skipCheck) => {
   let dumpData = {}
   const builtList = []
   let lastBuiltData = ''
   if (fs.existsSync(lastBuildsPath)) {
     lastBuiltData = await fs.readFile(lastBuildsPath, 'utf8')
   }
   for (const build of buildsList) {
-    const { outPath, actionBuildData, tempBuildDir, tempActionName } = build
-    const builtBefore = utils.actionBuiltBefore(lastBuiltData, actionBuildData)
+    const { outPath, buildHash, tempBuildDir, tempActionName, legacy, actionName } = build
+    const builtBefore = utils.actionBuiltBefore(lastBuiltData, buildHash)
     if (!builtBefore || skipCheck) {
-      dumpData = { ...dumpData, ...actionBuildData }
+      dumpData = { ...dumpData, ...buildHash }
       if (tempActionName) {
         // rename index.[contentHash] to index.js
         fs.renameSync(path.join(tempBuildDir, tempActionName), path.join(tempBuildDir, 'index.js'))
       }
       await utils.zip(tempBuildDir, outPath)
+      // create symlink for default package actions.
+      // this assure legacy support for the old output path.
+      if (legacy) {
+        const legacyActionBuild = path.join(distFolder, `${actionName}.zip`)
+        const legacyActionTemp = path.join(distFolder, `${actionName}-temp`)
+        fs.symlink(outPath, legacyActionBuild)
```

Comment:
@florind12 why do we need symlinks here?

Reply:
symlinks here were a mistake. they break windows as usually the tools do not have permission to create them.
[ACNA-1637](https://jira.corp.adobe.com/browse/ACNA-1637)